src/apiUtils.js

```diff
@@ -0,0 +1,42 @@
+
+/**
+ * Register an event handler for the given event name and for the given list of layers. Valid names are: `loaded`, `updated`.
+ *
+ * The 'loaded' event will be fired when all the layers are loaded (and their 'loaded' events are fired).
+ *
+ * The 'updated' event will be fired whenever one of the layers fired an 'updated event',
+ * but throttled by requestAnimationFrame to return a maximum of one event per frame.
+ *
+ * @param {string} eventName - Supported event
+ * @param {Layer[]} layerList
+ *
+ * @memberof carto
+ * @api
+ */
+export function on (eventName, layerList, callback) {
```

Comment:
The callback is called `n` times, being `n` the layerList length, and it should be called only once.

Reply:
After talking about this, this is working fine: the 'updated' event is called n times because it is the expected behaviour 👍